lefthook.yml

```diff
@@ -4,8 +4,8 @@ pre-commit:
     lint:
       files: git diff --name-only @{push}
       glob: "*.{js,ts,jsx,tsx}"
-      run: npx eslint {files}
+      run: yarn run linter
```

Comment:
- looks like linting script in `package.json` is `lint`, shouldn't this be `yarn run lint`